_posts/2025-04-28-visualizar-simulaciones-advance.md

```
@@ -0,0 +1,56 @@
+comments: false
+author: Catalina Pinilla
+---
+
+## Simulación
+
+### Visualizar simulaciones
+
+En Advance ahora es mucho más fácil ver las simulaciones de tus clientes, estas están en un solo lugar con toda la información que necesitas en una sola pantalla optimizando al máximo tu tiempo.
+
+<p align="center">
+  <img src="https://cdn.capitalexpress.cl/
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: img

(MD033, no-inline-html)